e2e/config.go

```diff
@@ -23,11 +32,10 @@ var LvmSubscriptionChannel string
 // DiskInstall indicates whether disks are needed to be installed.
 var DiskInstall bool
```

Comment:
Please recheck all these variables. The ones within the scope of the package should be in small case. Like `DiskInstall`.